resources/angery.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <color=Red><title>angery</title></color>
```

Comment:
You can't apply CSS to stuff in your `<head>`, and you should avoid using inline styles anyway.